src/main.rs

```diff
@@ -18,8 +18,25 @@ pub use pdu::PduEvent;
 pub use ruma_wrapper::{MatrixResult, Ruma};
 
 use rocket::{fairing::AdHoc, routes};
+use serde::{Deserialize};
+use directories::ProjectDirs;
+use std::fs::remove_dir_all;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "conduit", about = "Matrix Homeserver written in rust using rocket and ruma.")]
+pub struct CmdOptions {
+    /// Activate debug mode
+    // short and long flags (-d, --debug) will be deduced from the field's name
+    #[structopt(long, short)]
+    config: Option<String>,
+}
+
+
```

Comment:
I don't think we need the structopt dependency just for this. Can you remove this and just use std::env::args directly?